talktorials/1_ChEMBL/T1_ChEMBL.ipynb

```diff
@@ -10,7 +10,7 @@
     "\n",
```

Comment:
<p>If you want "aligned" results, check the <a href="https://docs.python.org/3/library/string.html#formatspec" target="_blank">formatting minilanguage</a> for space padding. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/TeachOpenCADD/pull/8/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='49'/>

Reply:
<p>Thanks, updated output accordingly.</p><p> </p>